### DISCUSSION TITLE
Requesting membership

### DISCUSSION THREAD

**Discussion by L0neGamer** [haskell-github-trust/.github#12] [General]
Hi all!
I'm a haskell dev interested in keeping things running around here. 

I'm currently a maintainer on discord-haskell (which has an active enough team that it won't be folded in), but I'm interested and happy enough to contribute to other projects as well. 

Let me know what I can do to help out!

**Comment by jamesdbrock** [comment]
Yeah you're definitely a capable Haskell programmer.

https://github.com/discord-haskell/discord-haskell/commits?author=L0neGamer

What's your real name?

**Comment by jamesdbrock** [comment]
And what's your Hackage account name?

https://hackage.haskell.org/users/

**Comment by L0neGamer** [comment]
My real name is Benjamin. 

https://hackage.haskell.org/user/BenjaminMcRae is my hackage user.

**Comment by jamesdbrock** [comment]
Hi Benjamin McRae, nice to meet you.

Because I don't personally know you well enough to trust you, and because you don't need to make any improvements to any **haskell-github-trust** packages right now, let's pause your membership request.

If in the future you want to make some specific improvements to some **haskell-github-trust** packages then let's talk more at that time.

**Reply by jamesdbrock** [reply]
And thanks for offering to help out.

**Reply by L0neGamer** [reply]
Not a worry, best of luck on the trust and I expect I'll see you around soon.

**Reply by L0neGamer** [reply]
Hi James! 

As of today, I'm the owner and a maintainer of ekg-json, which would be a good entry into the trust I feel.

**Reply by jamesdbrock** [reply]
Yeah I agree, ekg-json (and ekg-core) would be a good package for the Trust. I sent you an invite so you can add packages to the Trust if you want.

**Reply by jamesdbrock** [reply]
Welcome to Haskell Github Trust!

**Reply by L0neGamer** [reply]
Sorry James, I was busy in the last week, could I be reinvited?

**Reply by jamesdbrock** [reply]
Ok I re-invited you.
